docs/DeveloperGuide.adoc

```diff
@@ -346,6 +359,61 @@ We are using `java.util.logging` package for logging. The `LogsCenter` class is
 
 Certain properties of the application can be controlled (e.g App name, logging level) through the configuration file (default: `config.json`).
 
+=== Events
+
+We create an Events package in modle, to combine the event element such as name and venue into an event list. In the Commands package, we create an package named EventCommand, which the commands are stored in. We have totally four commands:
+*`addEventsCommand` -- Add a new event to the event list.
```

Comment:
You do not need to list down the commands here. It must be listed in the user guide. Perhaps, you can provide a sequence of possible actions for this feature similar to undo/redo feature